custom_fsync/sync.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+)
+
+func main() {
+	if len(os.Args) != 1 {
+		fmt.Fprintf(os.Stderr, "Usage: %s\n", os.Args[0])
+		os.Exit(1)
+	}
+
+	err := filepath.WalkDir(".", func(path string, di fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+
```

Comment:
do you need to handle directories explicitly here or is okay to fsync directories? do you need to sync the files within the directory before syncing the directory? what is the behavior when a directory is sync'd

Reply:
Fsyncing directories is fine. OFS FUSE will just receive fsync request on the directory node, but I don't think that OFS will sync files in the subdirs